packages/data/src/unwrap.ts

```diff
@@ -0,0 +1,3 @@
+
+export { ownedTokensThrowOnError as ownedTokens } from './api/ownedTokens/ownedTokens';
+export { tokensByStatusThrowOnError as tokensByStatus } from './api/tokensByStatus/tokensByStatus';
```

Comment:
Just want to quickly call attention to this one- we may want to extend this to other methods depending on the application.  Generally being able to consume the `{  data, error }` is nice in client apps for error display, but the ergonomics around one that simply throws the error or returns data is nice for most applications.

Reply:
To be clear, supporting both is the long term goal, see the update in README as well

Reply:
the idea around `{data,error} `was to have same pattern as Apollo and other libs have